DEVELOPMENT.md

```diff
@@ -0,0 +1,178 @@
+
+![The screen GitKraken shows when it first launches][i:GitKraken splash page]
+
+Once you've installed GitKraken, you're going to want to download the Buzz About code onto your computer. To do this, click the "Clone a repo" button on the left, and type in the url `https://github.com/mn-pollinators/buzz-about.git`. Now, you should be able to see the whole history of the project!
+
+![The screen GitKraken shows when you've opened a repo][i:GitKraken graph view]
+
+## Installing `npm`
+
+At this point, you should have all of the code for Buzz About on your computer, but you still don't have a way to run it. For that, you'll need to install `npm`, the one-stop shop for everything JavaScript.
+
+`npm` comes as part of the Node project, which you can download [here][Node.js]. The LTS (long-term support) version should be fine.
+
+On Windows, you'll be asked if you want to install tools for native modules. We don't have any native modules in this project, but you'll probably want to check that box anyway, in case you want to install any in the future. (Installing tools for native modules may take a while.)
+
+![The npm installer asking whether you'd like tools for native modules, with the checkbox checked][i:Windows, installing npm]
+
+## Telling `npm` how to talk to GitHub Packages
+
+GitHub Packages is where we keep all of the images for the website, along with some data about flowers and bees. 
+
+For `npm` read from GitHub Packages, you'll need to provide it with a personal access token.
+
+### Generating the personal access token
+
+You can create a personal access token by going to <https://github.com/settings/tokens> and clicking "Generate new token". Under "Scopes", you'll want to check `repo`, `read:packages`, `write:packages`, and `delete:packages`:
+
+![The "New personal access token" page on GitHub, with those four scopes checked.][i:New personal access token]
+
+Then, you can click "Generate token" at the bottom of the page. You'll get a token that looks like `745b0dcc742dedf87c67ff075f65eb87295e205f`. Keep that around.
+
+### Giving that personal access token to `npm`
+
+Next, open a command prompt (Terminal.app on the Mac; PowerShell on Windows) and run
+
+```sh
+npm login --registry=https://npm.pkg.github.com
+```
+
+ * When it asks for a username, enter your GitHub username.
+ * When it asks for a password, paste in the personal access token (***not*** your GitHub password).
+ * When it asks for your email, enter your GitHub email. 
+
+![Powershell after running "npm login --registry=https://npm.pkg.github.com". The output "Logged in as helloworld12321demo on https://npm.pkg.github.com/" is shown.][i:Logging in to npm]
+
+Some sticking-points:
+
+1. For security reasons, what you type into the "password" field is invisible.
+2. To paste text into PowerShell, right-click once.
+3. On github.com/settings/tokens, there personal access token has a space after it&mdash;be careful that you don't copy that! :upside_down_face:
```

Comment:
```suggestion
3. On github.com/settings/tokens, the personal access token has a space after it&mdash;be careful that you don't copy that! :upside_down_face:
```

Also, this is why there is a copy button right there.

Reply:
Ha, that's true—I just thought I should warn those people who think "Hmph! I don't need some button to copy text for me! I'll just do it myself :triumph:" only to have the access token get rejected because they didn't copy it right.

Not that I know anyone who would do that&hellip;